README.md

```diff
@@ -67,6 +67,9 @@ In your pyproject.toml file, you may configure the following options:
   * `metadata`: Boolean. Default: unset. If true, include the commit hash in
     the version, and also include a dirty flag if `dirty` is true. If unset,
     metadata will only be included if you are on a commit without a version tag.
+  * `tagged_metadata`: Boolean. Default: false. If true, include the tagged metadata
+    discovered as the first of the metadata segments.  Requires metatadata to be
+    true as well to have effect.
```

Comment:
It'll also work if `metadata` is left unset (`None`, the default), so:

> Has no effect when `metadata` is false.

But that's a minor detail, so I can take care of it when I'm updating the changelog.